webnext/addon.py

```diff
@@ -0,0 +1,63 @@
+"""
+This mitmproxy serves the production build of the mitmwebnext interface and strips the Origin header from all outgoing requests to the mitmweb API.
```

Comment:
What's the purpose of this addon? I understand what it does, but I don't understand why/what we want this for. 😄